dockerenforcer/killer.py

```diff
@@ -96,6 +104,19 @@ def should_be_killed(self, container):
                 logger.error("During execution of rule {0} exception was raised: {1}".format(rule['name'], e))
         return Verdict(False, container, None)
 
+    def all_violated_rules(self, container):
+        if not container:
+            return []
+
+        v_rules = []
+        for rule in self.__rules:
+            try:
+                if rule['rule'](container):
+                    v_rules.append(rule['name'])
```

Comment:
I would add a new option `STOP_ON_FIRST_VIOLATION`. If it's `True`, then we run all check for every container and record all violations and display all violation. If it's `False`, we stop on the first one and never check the others.